keda/values.yaml

```diff
@@ -376,7 +375,7 @@ prometheus:
       targetPort:
       interval:
       scrapeTimeout:
-      relabellings: []
+      relabelings: []
```

Comment:
This is kinda breaking. But if we accept this, this change needs to be reflected into the 3 servicemonitor.yaml files:
- templates/manager/servicemonitor.yaml
- templates/webhooks/servicemonitor.yaml
- templates/metrics-server/servicemonitor.yaml

Reply:
Agreed, we should not break this